content/docs/command-reference/exp/remove.md

```diff
@@ -57,5 +58,24 @@ $ dvc exp remove exp-1dad0 exp-1df77
 
 $ dvc exp list
 master:
+        exp-9fcef
         exp-e6c97
 ```
+
+We can also remove all of the experiments from a remote Git repository:
+
+```dvc
+$ dvc exp push myremote exp-e6c97
+Pushed experiment 'exp-e6c97'to Git remote 'myremote'.
+$ dvc exp push myremote exp-9fcef
+Pushed experiment 'exp-9fcef'to Git remote 'myremote'.
+
+$ dvc exp list myremote
+master:
+        exp-aaa23
+        exp-e6c97
+
+$ dvc exp remote -r myremote
+$ dvc exp list myremote
+$
```

Comment:
remove this extra line?

Reply:
Yeah or leave it blank but it may seem a bit strange. Not clear that nothing is printed because all the experiments were removed...
```suggestion

```

Reply:
Can there be more than one remote listed by `list`? If so showing 2 git remotes after deleting from one only could help here.